src/platforms/javascript/guides/nextjs/manual-setup/run-dev.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Running in development
+sidebar_order: 4
+description: "Learn how to enable the SDK to run in development mode."
+---
+
+Next.js’ development mode doesn’t perform some checks the production mode does during the build process, and there may be errors that aren’t caught (including these that would prevent a successful build) and make the SDK not behave as expected.
+Thus, the Next.js SDK is only enabled by default in production environments.
```

Comment:
```suggestion
Sentry's Next.js SDK is enabled by default in production environments. This default mode is because Next.js’ development mode doesn’t perform some checks as it does in production mode during the build process. As a result, some errors may not be caught (including these that would prevent a successful build) and the SDK may not behave as expected.
```